lib/src/widgets/chat/ds_location_message_bubble.widget.dart

```diff
@@ -60,13 +61,24 @@ class DSLocationMessageBubble extends StatelessWidget {
           mainAxisSize: MainAxisSize.min,
           crossAxisAlignment: CrossAxisAlignment.center,
           children: [
-            DSCachedNetworkImageView(
-              url:
-                  'https://maps.googleapis.com/maps/api/staticmap?&size=360x360&markers=$latitude,$longitude&key=${DSAuthService.googleKey}',
-              placeholder: (_, __) => _buildLoading(),
-              align: align,
-              style: style,
-            ),
+            double.tryParse(latitude) != null &&
+                    double.tryParse(longitude) != null
```

Comment:
Your implementation works fine, but since this condition is exactly same as the ```if(lat != null && long != null)``` done in ```onTap```, I would do the following changes to improve readability and maintainability:

1.  Do this lat and long parse outside of the ```onTap``` (I'd do it right after the ```foregroundColor``` assign in ```build``` method). 
2. Create a variable named like ```hasValidCoordinates``` to store the ```lat != null && long != null``` result.
3. Pass a method to ```onTap``` only if ```hasValidCoordinates``` is true, otherwise ```onTap``` should be ```null```.
4. Use ```hasValidCoordinates``` to do the validation you just did here.